doc/sphinx/source/data/commondata.rst

```diff
@@ -0,0 +1,355 @@
+This allows the user the flexibility of creating variants
+where only a subset of the uncertainties are modified.
+
+The format of the uncertainty files is of two fields, a ``definitions`` field that contains
+metadata about all the uncertainties: name, treatment (``ADD`` or ``MULT``) and type
+and a second field ``bins`` which is a list of mappings with ``ndata`` entries
+with the named uncertainties.
+
+Note that, regardless of their treatment, uncertainties should always be written as absolute values
+and not relative to the data values. If the data should be updated, the uncertainties should be too.
+
+..  code-block:: yaml
+
+    definitions:
+        stat:
+            description:
+            treatment:
+            type:
+        error_name:
+            description:
+            treatment:
+            type:
+        error_name_2:
+            description:
+            treatment:
+            type:
+    bins:
+        - stat:
+          error_name:
+          error_name_2:
+
+
+
+
+``variants``
+~~~~~~~~~~~~
+
+In some occasions we might want to maintain two variations of the same observable.
+For instance, we might have two incompatible sources of uncertainties. In such case a variant can be added.
+These variants can overwrite certain keys if necessary.
+When a variant is used, the key under the variant will be used instead of the key defined in the observable.
+
+A ``variant`` can only overwrite the entries ``data_central``, ``theory`` and ``data_uncertainties``.
+Example:
+
+..  code-block:: yaml
+
+    data_uncertainties:
+        - uncertainties.yaml
+
+    variants:
+        name_of_the_variant:
+            data_uncertainties:
+                - uncertainties.yaml
+                - extra_uncertainties.yaml
+        another_variant:
+            data_central: different_data.yaml
+            data_uncertainties:
+                - different_uncertainties.yaml
+              
+When loading this dataset with no variant only the ``uncertainties.yaml`` file will be read.
+Instead, when choosing ``variant: name_of_the_variant``, both ``uncertainties.yaml`` and  ``extra_uncertainties.yaml`` will be loaded.
+If we select ``variant: another_variant`` both the ``data_uncertainties`` and the ``data_central`` keys will be substituted.
+Note that if we want to substitute the default set of uncertainties we just need to not include it in the variant (as done in ``another_variant``).
+
+``theory``
+~~~~~~~~~~
+
+The theory field defines how predictions for the dataset are to be computed.
+It includes two entries:
+
+- ``FK_tables``: this is a list of lists which defines the FK Tables to be loaded. The outermost list are the operands (in case an operation is needed to recover the observable, more on that below). The innermost list are the grids that are to be concatenated in order to form the operands.
+- ``operaton``: operation to be applied in order to compute the observable
```

Comment:
```suggestion
- ``operation``: operation to be applied in order to compute the observable
```
"vp currently supports 'ratio', 'asy', ...." "The identity transformation is indicated by None." or similar